### ISSUE TITLE
Medication examples using STU3 codesystem URL for Medication.code medication-type extension

### ISSUE THREAD

**Issue by robeastwood-agency** [hl7au/au-fhir-base#604]
The QA report lists numerous errors related to medication examples using the old STU3 URL for the medication-type CodeSystem.

STU3 URL: http://hl7.org.au/fhir/CodeSystem/medication-type

R4 URL: http://terminology.hl7.org.au/CodeSystem/medication-type

The AU Medication Type extension is bound to the R4 ValueSet using the corresponding R4 URL.

Medication examples have similar to the following, with the old STU3 system URL:
```
<valueCoding>
     <system value="http://hl7.org.au/fhir/CodeSystem/medication-type"/>
     <code value="BPD"/>
     <display value="Branded product with no strengths or form"/>
</valueCoding>
```
This affects 16 examples.

**Comment by jaymeemurdoch** [comment]
Pull request is forthcoming for this fix.

**Comment by jaymeemurdoch** [comment]
Amended URL from STU3 (http://hl7.org.au/fhir/CodeSystem/medication-type) to R4 (http://terminology.hl7.org.au/CodeSystem/medication-type) in the 16 examples.

Confirmed fix with numerous QA errors on local build resolved, for example Bundle/9309d080-4e97-11e9-8647-d663bd873d93.
